components/src/StatusIndicator/StatusIndicator.js

```diff
@@ -0,0 +1,88 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { space } from "styled-system";
+import theme from "../theme";
+
+const StatusIndicatorColours = {
+  neutral: {
+    borderColor: theme.colors.lightGrey,
+    backgroundColor: theme.colors.lightGrey,
+    color: theme.colors.darkGrey
+  },
+  quiet: {
+    borderColor: theme.colors.white,
+    backgroundColor: theme.colors.white,
+    color: theme.colors.darkGrey
+  },
+  danger: {
+    borderColor: theme.colors.red,
+    backgroundColor: theme.colors.red,
+    color: theme.colors.white
+  },
+  informative: {
+    borderColor: theme.colors.blue,
+    backgroundColor: theme.colors.blue,
+    color: theme.colors.white
+  },
+  success: {
+    borderColor: theme.colors.green,
+    backgroundColor: theme.colors.green,
+    color: theme.colors.white
+  },
+  warning: {
+    borderColor: theme.colors.yellow,
+    backgroundColor: theme.colors.yellow,
+    color: theme.colors.darkGrey
+  }
+};
+
+const getStatusIndicatorColours = props => {
+  if (props.type === "neutral") {
+    return StatusIndicatorColours.neutral;
```

Comment:
could use a switch here, or create an object mapping type to colors {
neutral: StatusIndicatorColours.neutral
}